### PR TITLE
AWS RDS: report resolved issue

### PR DESCRIPTION
**Pull request by zucchinidev**
[#183452044](https://www.pivotaltracker.com/story/show/183452044)

Which other branches should this be merged with (if any)?

None.

### CODE REVIEW

**Comment by zucchinidev** [comment]
@FelisiaM You would be so kind as to review this change, it is the first time I report an incident. Please tell me if I am sending the right message.

**Comment by RichardJJG** [comment]
@zucchinidev, remove this from draft mode when it's ready to merge, please. It's okay to use common contractions in the docs, by the way (`wasn't` does not need to be `was not`)

**Comment by RichardJJG** [comment]
@zucchinidev, that's okay. You might want to switch to a Markdown linter in the IDE when looking at docs -- it will make them easier to read.

File: release-notes.html.md.erb [hunk 1]
```diff
@@ -124,7 +124,10 @@ New features and changes in this release:
 
 This release has the following fixes:
 
-- **Run-in heading:** Description
+- **Correct behaviour of the AWS RDS security configuration to avoid deletion of the database instance:** This configuration
```
Comment (FelisiaM) [inline release-notes.html.md.erb:127]: I think there is maybe a more concise way to express what the fix is to the customers. 
The current summary describes all the scenario how to reproduce it and it includes a part that has not changed and its expected behaviour. 

Maybe saying something like: 
Disabling deletion protection will result in successful future deletion operations. Previously when the `deletion_protection` property was turned off after a failed deletion attempt, the subsequent deletes would remove the instance from CF, but not from the IaaS.

**Comment by zucchinidev** [comment]
@RichardJJG thank you very much for your advice. I work much better with the Markdown linter. The PR is ready to review.

**Comment by RichardJJG** [comment]
@zucchinidev, could you now resolve the merge conflict, please?

**Comment by zucchinidev** [comment]
Done!! Thank you very much @RichardJJG!!

**Pull request merged**